events/apigw.go

```diff
@@ -133,63 +133,65 @@ type APIGatewayV2HTTPResponse struct {
+	CognitoAuthenticationType     string `json:"cognitoAuthenticationType,omitempty"`
+	CognitoAuthenticationProvider string `json:"cognitoAuthenticationProvider,omitempty"`
+	UserArn                       string `json:"userArn,omitempty"` //nolint: stylecheck
 	UserAgent                     string `json:"userAgent"`
-	User                          string `json:"user"`
+	User                          string `json:"user,omitempty"`
 }
 
 // APIGatewayWebsocketProxyRequest contains data coming from the API Gateway proxy
 type APIGatewayWebsocketProxyRequest struct {
-	Resource                        string                                 `json:"resource"` // The resource path defined in API Gateway
-	Path                            string                                 `json:"path"`     // The url path for the caller
+	Resource                        string                                 `json:"resource,omitempty"` // The resource path defined in API Gateway
+	Path                            string                                 `json:"path,omitempty"`     // The url path for the caller
 	HTTPMethod                      string                                 `json:"httpMethod,omitempty"`
-	Headers                         map[string]string                      `json:"headers"`
-	MultiValueHeaders               map[string][]string                    `json:"multiValueHeaders"`
-	QueryStringParameters           map[string]string                      `json:"queryStringParameters"`
-	MultiValueQueryStringParameters map[string][]string                    `json:"multiValueQueryStringParameters"`
-	PathParameters                  map[string]string                      `json:"pathParameters"`
-	StageVariables                  map[string]string                      `json:"stageVariables"`
+	Headers                         map[string]string                      `json:"headers,omitempty"`
+	MultiValueHeaders               map[string][]string                    `json:"multiValueHeaders,omitempty"`
+	QueryStringParameters           map[string]string                      `json:"queryStringParameters,omitempty"`
+	MultiValueQueryStringParameters map[string][]string                    `json:"multiValueQueryStringParameters,omitempty"`
+	PathParameters                  map[string]string                      `json:"pathParameters,omitempty"`
+	StageVariables                  map[string]string                      `json:"stageVariables,omitempty"`
 	RequestContext                  APIGatewayWebsocketProxyRequestContext `json:"requestContext"`
-	Body                            string                                 `json:"body"`
-	IsBase64Encoded                 bool                                   `json:"isBase64Encoded,omitempty"`
+	Body                            string                                 `json:"body,omitempty"`
+	IsBase64Encoded                 *bool                                  `json:"isBase64Encoded,omitempty"`
```

Comment:
breaks back-compat, revert back to `bool`

Reply:
Reverted the change. 🙇